reviews/sandbox/2023/2023-telepresence-annual.md

```diff
@@ -0,0 +1,91 @@
+# Telepresence Sandbox Annual Review
+
+The goal of the Telepresence project is to accelerate the "developer inner loop" for cloud-native application development on Kubernetes. Telepresence achieves this by creating a dynamic network bridge between a local development environment (e.g., a laptop) and a remote Kubernetes cluster.
+
+Telepresence was presented to the TOC on April 2018. Please see the project's [Sandbox proposal](https://github.com/cncf/toc/blob/master/proposals/sandbox/telepresence.adoc).
+
+## Annual Review Questions
+
+### Metrics
+
+| Metric       |  Pre-Sandbox |     2020     |    2021      |   2022    |   2023   |
+| ------------ | ------------ | ------------ | ------------ | --------- | -------- |
+| Docker Pulls | 100K+        | 10M+         | 10M+         | 10M+      | 10M+     |
+| Stars        | 700+         | 2700+        | 4000         | 5.1k      | 5.8K     |
+| Forks        | ~50          | ~190         | ~300         | 400+      | 400+     |
+
+
+The CNCF provides Telepresence Github metrics through its [devstats site](https://telepresence.devstats.cncf.io/d/8/dashboards?orgId=1&refresh=15m).
+
+### Maintainers
+
+There are eight maintainers total. Of these, four commit to the project on a daily basis and are employed by Ambassador Labs.
+The remaining four are three Ambassador Labs employees and one external maintainer, all of whom commit less frequently as needed.
+
+### Adoption
+
+The Telepresence project has seen continued growth and adoption since entering the CNCF Sandbox. In addition to the adopters mentioned last year:
+
+* Avaya
+* Bose
+* Engel & Volkers (KubeCon EU 2019 presentation: https://www.youtube.com/watch?v=tD0FIlxO1AQ)
+* InVision
+* Namely
+* PTC
+* Red Hat
+* Timecampus
+* ThetaRay
+
+We have added new adopters such as:
+
+* Salt Security
+* MoneyForward
+* NetApp
+* Manhattan Associates
+* SKU IQ
+
+
+There have also been a number of integrations with other software stacks including:
+
+* wad.sh (https://blog.sebastian-daschner.com/entries/reloading-apps-kubernetes-telepresence-wad)
+* Ambassador Edge Stack (https://www.getambassador.io/docs/latest/topics/using/edgectl/)
+* Kelda (https://kelda.io/docs/tutorials/telepresence/)
+* Ambassador Cloud/Developer Control Pane (https://www.getambassador.io/)
+
+From a user standpoint, we have been engaging with the Telepresence community and growing the user base. The #telepresence slack channel has around 7700 members now and continues to be a great way for us to keep in touch with the community and receive feedback. We hold a live troubleshoting session through the channel weekly and usually work through 1-2 users issues. We also hold a monthly Maintainers meeting attended by our development team and open to anyone to ask questions, discuss features, etc.
```

Comment:
The Slack channel is still linked to the vendor this should be in a CNCF owned Slack https://datawire-oss.slack.com/join/shared_invite/zt-2606kjsn6-Xr2gP_ks9TieFsTV_m3cGQ#/shared-invite/email

I can't find recordings of the maintainers meetings.

Reply:
Migrating to a CNCF owned Slack is something we want to do and have been looking into.

Do you have any guidance that can help us with this and ensure a smooth transition?  Is the correct path to getting the channel created just submitting a request in to the help desk as mention [here](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/attend/slack-guidelines/)?

Any additional information would be greatly appreciated.